### DISCUSSION TITLE
event to action

### DISCUSSION THREAD

**Discussion by Tihomir971** [sveltejs/kit#7849] [Q&A]
I have dispatched event from child component. Can I call action with that event?

**Comment by david-plugge** [comment]
Can you give an example? Is it about manually invoking an action without a form?

**Reply by Tihomir971** [reply]
Yes, I need to call function from form action without form. I don't want to duplicate functions

**Reply by david-plugge** [reply]
Take a look at this page in the docs:
https://kit.svelte.dev/docs/form-actions#progressive-enhancement-custom-event-listener

You can create your own function that calls the action using fetch

**Comment by teemingc** [comment]
I think you'll have to `fetch` the form action URL since it lives on the server.
